Gemfile

```
@@ -5,9 +5,13 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+gem 'parser', '~>3.2.2.3'
 gem "rails_autolink"
 gem "acts_as_votable", "~> 0.14.0"
-gem "aws-sdk-rails"
+#gem "aws-sdk-rails"
+gem "aws-sdk-ses"
+#gem "aws-sdk"
+gem 'stringio', '~> 3.1', '>= 3.1.2'
```

rubocop (1.69.1)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)